datatableview/columns.py

```diff
@@ -11,7 +11,7 @@
 import django
 from django.db import models
 from django.db.models import Model, Manager, Q
-from django.db.models.fields import FieldDoesNotExist
+from django.core.exceptions import FieldDoesNotExist
 from django.core.exceptions import ObjectDoesNotExist
```

Comment:
Could be one the same line
`from django.core.exceptions import FieldDoesNotExist, ObjectDoesNotExist`

Reply:
Good find, will update 👍